patcher.py:532: UserWarning: WARNING: One or more of your selected dataset(s) contains at least one netcdf file that does not follow netcdf convections. Less useful netcdf loader had to be used.
  warnings.warn('WARNING: One or more of your selected dataset(s) contains at least one netcdf file that does not follow netcdf convections. Less useful netcdf loader had to be used.')
Traceback (most recent call last):
  File "patcher.py", line 1575, in <module>
    patcher.run()
  File "patcher.py", line 406, in run
    self._load_datasets_from_disk()
  File "patcher.py", line 747, in _load_datasets_from_disk
    self.master_xarray_dataset_examples = self._create_master_xarray_dataset(loaded_datasets_examples, loaded_datetimes_examples, dataset_configs_examples)
  File "patcher.py", line 850, in _create_master_xarray_dataset
    master_xarray_dataset = master_xarray_dataset.assign(xarray_dataset_datetimes)
UnboundLocalError: local variable 'master_xarray_dataset' referenced before assignment
slurmstepd: error: acct_gather_profile/influxdb _send_data: curl_easy_perform failed to send data (discarded). Reason: Couldn't resolve host name
